src/gulpfile.babel.js

```diff
@@ -143,6 +144,63 @@ gulp.task( 'styles', () => {
 		.pipe( notify({ message: '\n\n✅  ===> STYLES — completed!\n', onLast: true }) );
 });
 
+/**
+ * Task: `addonStyles`.
+ *
+ * Compiles Sass, Autoprefixes it and Minifies CSS.
+ *
+ * This task does the following:
+ *    1. Gets the source scss file
+ *    2. Compiles Sass to CSS
+ *    3. Writes Sourcemaps for it
+ *    4. Autoprefixes it and generates CSS file
+ *    5. Renames the CSS file with suffix .min.css
+ *    6. Minifies the CSS file and generates .min.css
+ *    7. Injects CSS or reloads the browser via browserSync
+ */
+gulp.task( 'addonStyles', ( done ) => {
+	// Exit task when no addon styles
+	if ( config.addonStyles.length === 0 ) {
+		return done();
+	}
+
+	// Process each addon style
+	var tasks = config.addonStyles.map( function ( addon ) {
+
+		return gulp
+			.src( addon.styleSRC, { allowEmpty: true })
```

Comment:
I wish there was less repetition. This styles task is repeated four times now, maybe we can make a function that can be used everywhere in styles, stylesRTL and addonStyles/RTL tasks? Would you be up for it?